### DISCUSSION TITLE
[Feature] [Nodejs] Surport prisma ORM

### DISCUSSION THREAD

**Discussion by mage520** [apache/skywalking#8527] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

[skywalking-nodejs](https://github.com/apache/skywalking-nodejs)/[src](https://github.com/apache/skywalking-nodejs/tree/master/src)/plugins/

add prisma plugin

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
As you don't check the will contribute, this would be a discussion only.
Plugins are various and countless, we wouldn't track this as this kind of feature requirements would overload the issue tracker.
